src/components/BigButton/BigButton.style.scss

```diff
@@ -0,0 +1,10 @@
+.bigbutton {
+  background: transparent;
```

Comment:
could you please sort props `asc` ?
while we decide a methodology to implement in the project it would be nice to have class names separated by hyphens

Reply:
Done.